custom_metrics/privacy.js

```diff
@@ -1,24 +1,312 @@
-    var pattern = new RegExp(words, 'i');
+    var pattern = new RegExp('\\b(?:' + words + ')\\b', 'i');
+
+    return Array.from(document.querySelectorAll('a')).filter(a => a.innerText.match(pattern))
+      .length;
```

Comment:
Would it be useful to output which privacy wording was used? It may facilitate debugging later or provide interesting data to analyze about which words are more popular.